index.html

```diff
@@ -2030,6 +2030,100 @@ <h4>Presentations Using Derived Credentials</h4>
+          </p>
+          <p>
+A <a>verifiable presentation</a> that includes a self-asserted
+<a>verifiable credential</a> that is only secured using the same mechanism as
+the <a>verifiable presentation</a> MUST include a <code>holder</code>
+<a>property</a>.
+          </p>
+          <p>
+All of the normative requirements defined for <a>verifiable credentials</a>
+apply to self-asserted <a>verifiable credentials</a>.
+          </p>
+          <p>
+For self-asserted <a>verifiable credentials</a> that are secured using the same
+mechanism as the <a>verifiable presentation</a>, the value of the
+<code>issuer</code> <a>property</a> MUST be identical to the <code>holder</code>
+<a>property</a> of the <a>verifiable presentation</a>.
+          </p>
+          <p>
+The example below shows a <a>verifiable presentation</a> that embeds a
+self-asserted <a>verifiable credential</a> that is secured using the same
+mechanism as the <a>verifiable presentation</a>.
+          </p>
+
+          <pre class="example nohighlight" title="A secured verifiable presentation relying on an embedded proof, including a self-asserted verifiable credential">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "type": ["VerifiablePresentation", "ExamplePresentation"],
+  "holder": "did:example:12345678",
+  "verifiableCredential": [{
+    "@context": "https://www.w3.org/ns/credentials/v2",
+    "type": ["VerifiableCredential", "SelfAssertedCredential", "ExampleFoodPreferenceCredential"],
+    "issuer": "did:example:12345678",
+    "credentialSubject": {
+      "favoriteCheese": "Gouda"
+    },
+    { <span class="comment">...</span> }
+  }],
+  "proof": [{ <span class="comment">...</span> }]</span>
+}
+          </pre>
+          <p>
+The example below shows a <a>verifiable presentation</a> that embeds a
+self-asserted <a>verifiable credential</a> that holds <a>claims</a> about the
+<a>verifiable presentation</a>. It is secured using the same mechanism as the
+<a>verifiable presentation</a>.
+          </p>
+
+          <pre class="example nohighlight" title="A secured verifiable presentation, relying on an embedded proof, with a self-asserted verifiable credential about the verifiable presentation">
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
+  ],
+  "type": ["VerifiablePresentation", "ExamplePresentation"],
+  <span class="highlight">"id": "http://example.com/presentation/5223"</span>,
+  "holder": "did:example:12345678",
+  "verifiableCredential": [{
+    "@context": "https://www.w3.org/ns/credentials/v2",
+    "type": ["VerifiableCredential", "ExampleTimeLimitCredential"],
+    "issuer": "did:example:12345678",
+    "credentialSubject": {
+      <span class="highlight">"id": "http://example.com/presentation/5223"</span>,
+      "validUntil": "2024-01-01T19:23:24Z"
```

Comment:
```suggestion
      "validUntil": "2024-01-01T19:23:24Z"
```

Huh... this is valid, and begs the question... why not just put it on the presentation itself? We might want to have an example of someone putting a self-asserted claim on the VP itself... possibly in another PR, we don't want to complicate this one further.

IOW, do we want to enable `validFrom` and `validUntil` onto the Presentation too? I think we do want to enable that. At present, it's only allowed on a `VerifiableCredential`. Historically, this was because presentations were expected to always be short lived... but the short-lived-ness being determined by the verifier and the `created/iat` date on the proof.

Reply:
Agreed, we don't want to derail this PR with trying to find the perfect example. I defer to @jandrieu to suggest a use case. I'll try to think of a use case (with markup) as well.

Reply:
I've updated the example based on Joe's comment here: https://github.com/w3c/vc-data-model/pull/1186/files#r1268305835